src/System.Runtime.Intrinsics/ref/System.Runtime.Intrinsics.cs

```diff
@@ -44,9 +44,11 @@ public static class Vector64
         public static Vector64<uint> CreateScalarUnsafe(uint value) { throw null; }
     }
     [StructLayout(LayoutKind.Sequential, Size = 8)]
-    public readonly struct Vector64<T> where T : struct
+    public readonly struct Vector64<T> : IEquatable<Vector64<T>>, IFormattable
+        where T : struct
     {
         private readonly int _dummy;
+        public static int Count { get { throw null; } }
```

Comment:
Do you plan to add tests for these newly exposed APIs?

Reply:
They will be added in CoreCLR with the rest of the intrinsic tests.